### PR TITLE
Added product page

### PR DESCRIPTION
**Pull request by dubeyaditya29**
Added product display page.
Screenshot (84)

### CODE REVIEW

**Review by Shivanshsharma01 — APPROVED** [review]
Clean code with good use of variables. Good to merge

File: styles/pdp.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  align-items: center;
+  justify-content: center;
+}
+
+.main-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 10%;
```
Comment (RakshithBhat03) [inline styles/pdp.css]: You can remove this line. Margin top is getting applied in the next line anyway.

**Review by RakshithBhat03 — APPROVED** [review]
Everything looks good. The code is clean and well structured. Good to merge

File: styles/pdp.css (hunk 1 above)
Comment (dubeyaditya29) [reply styles/pdp.css]: yeah have removed it thanks for pointing out.

**Pull request merged**